docker/wait-for-dependencies-postgres.sh

```diff
@@ -10,6 +10,7 @@ done
 TMP=$PROXY_HOST
 TMP2=$AMQP_BROKER_HOST
 TMP3=$DB_HOST
+TMP4=$INFLUXDB_HOST
```

Comment:
Why is this needed?

Reply:
I'll try to keep this short but it's a rather involved answer. In my initial push the rpm build worked fine because the influxdb client was able to connect to the host over 127.0.0.1. The docker build, it turns out, was also using 127.0.0.1. To resolve this, I added a variable to hold the value of the INFLUXDB_HOST on both the iml-stats and iml-device services. This was added to the docker-compose file as an environment variable and it fixed the issue for docker. But then I noticed the RPM build was broken and it turns out it broke because the INFLUXDB_HOST variable needed to be included in the print-settings output in order for the rpm service to load the environment variable. So I added it to the print-settings list and sure enough that fixed the rpm build. But now, the docker build was broke again... While inspecting the docker containers I saw that the INFLUXDB_HOST variable was set as expected but looking at /proc/1/environ it was apparent that the INFLUXDB_HOST was 127.0.0.1. I then realized that the environment variables in the settings file will overwrite the environment variables specified in the docker-compose file. To resolve this, I needed to update`wait-for-dependencies.sh` because it was loading the settings file and then overwriting specific environment variables for docker, such as PROXY_HOST and DB_HOST. Now, INFLUXDB_HOST is overwritten in the wait-for-settings file AFTER the settings file is loaded. If this is not done, the docker container will not retain the value that is set in the docker-compose environment block.

Reply:
We've discussed a new solution. Resolving this conversation for now.